ram/wram0.asm

```diff
@@ -197,6 +197,9 @@ wLinkOtherPlayerGender:: db
 
 wPalFlags:: db
 
+wCurrentWeather:: db
```

Comment:
Why are `wCurrentWeather` and `wPrevWeather` separated? (Also, why not `wCurWeather` to match?)